2023-08-18 10:52:04,588 INFO    MainThread:47163 [internal.py:wandb_internal():90] W&B internal server running at pid: 47163, started at: 2023-08-18 10:52:04.588061
2023-08-18 10:52:04,589 DEBUG   HandlerThread:47163 [handler.py:handle_request():141] handle_request: run_start
2023-08-18 10:52:04,589 INFO    WriterThread:47163 [datastore.py:open_for_write():75] open: /home/nicole/marl-emecom/src/experiments_anastassacos/wandb/offline-run-20230818_105204-1t3yxl97/run-1t3yxl97.wandb
2023-08-18 10:52:04,591 DEBUG   HandlerThread:47163 [meta.py:__init__():35] meta init
2023-08-18 10:52:04,591 DEBUG   HandlerThread:47163 [meta.py:__init__():49] meta init done
2023-08-18 10:52:04,591 DEBUG   HandlerThread:47163 [meta.py:probe():209] probe
2023-08-18 10:52:04,592 DEBUG   HandlerThread:47163 [meta.py:_setup_git():199] setup git
2023-08-18 10:52:04,600 DEBUG   HandlerThread:47163 [meta.py:_setup_git():206] setup git done
2023-08-18 10:52:04,600 DEBUG   HandlerThread:47163 [meta.py:_save_pip():53] save pip
2023-08-18 10:52:04,600 DEBUG   HandlerThread:47163 [meta.py:_save_pip():67] save pip done
2023-08-18 10:52:04,600 DEBUG   HandlerThread:47163 [meta.py:probe():247] probe done
2023-08-18 10:52:26,509 WARNING MainThread:47163 [internal.py:wandb_internal():153] Internal process interrupt: 1
2023-08-18 10:52:26,733 WARNING MainThread:47163 [internal.py:wandb_internal():153] Internal process interrupt: 2
2023-08-18 10:52:26,734 ERROR   MainThread:47163 [internal.py:wandb_internal():156] Internal process interrupted.
2023-08-18 10:52:27,512 INFO    WriterThread:47163 [datastore.py:close():279] close: /home/nicole/marl-emecom/src/experiments_anastassacos/wandb/offline-run-20230818_105204-1t3yxl97/run-1t3yxl97.wandb
2023-08-18 10:52:27,513 INFO    HandlerThread:47163 [handler.py:finish():806] shutting down handler
2023-08-18 10:52:27,595 INFO    SenderThread:47163 [sender.py:finish():1111] shutting down sender
2023-08-18 10:52:27,596 INFO    MainThread:47163 [internal.py:handle_exit():80] Internal process exited
